Library/Formula/zmap.rb

```diff
@@ -1,41 +1,44 @@
 class Zmap < Formula
   desc "Network scanner for Internet-wide network studies"
   homepage "https://zmap.io"
-  url "https://github.com/zmap/zmap/archive/v1.2.1.tar.gz"
-  sha256 "ff01f2b139e98a7e64ed2872ae332929c500dd6120888d57900fac82dccfeea8"
+  url "https://github.com/zmap/zmap/archive/v2.1.1.tar.gz"
+  sha256 "29627520c81101de01b0213434adb218a9f1210bfd3f2dcfdfc1f975dbce6399"
 
   head "https://github.com/zmap/zmap.git"
 
   bottle do
     cellar :any
-    sha1 "67ee1c3239a4109088a7b5f9d9db4884640c1920" => :mavericks
-    sha1 "3d780dc9a1125d96294d7071b1cdc359ccf4f302" => :mountain_lion
-    sha1 "b0482d7fedc3063f7218997f4ecdcb93043f898a" => :lion
+    sha256 "829eb6548654b367cffb7eefeaf04ed41425e97c4f276043c0a6bba467a2d68e" => :yosemite
```

Comment:
Please don't add or remove bottle checksums. We'll do that automatically on pull.